### ISSUE TITLE
Xvnc: ../../include/privates.h:121: dixGetPrivateAddr: Assertion `key->initialized' failed.

### ISSUE THREAD

**Issue by ne0YT** [kasmtech/KasmVNC#172]
**Describe the bug**
I ran sudo in my KasmVNC session and it immidiately crashed.
since then I cannot start vncserver anymore with this error log:

```
Xvnc KasmVNC 1.1.0.56c840fc947d891f054bf251d2a02454d3e6f686 - built Apr  5 2023 14:38:59
Copyright (C) 1999-2018 KasmVNC Team and many others (see README.me)
See http://kasmweb.com for information on KasmVNC.
Underlying X server release 12008000, The X.Org Foundation

 2023-08-14 12:13:57,236 [DEBUG] vncext: Adding DLP binary mime type chromium/x-web-custom-data
 2023-08-14 12:13:57,236 [DEBUG] vncext: Adding DLP binary mime type text/html
 2023-08-14 12:13:57,236 [DEBUG] vncext: Adding DLP binary mime type image/png
 2023-08-14 12:13:57,236 [DEBUG] vncext: Total 3 binary mime types
 2023-08-14 12:13:57,236 [INFO] ICE: Querying public IP...
 2023-08-14 12:13:57,236 [DEBUG] ICE: Trying 'stun.l.google.com', port 19302
 2023-08-14 12:13:57,255 [DEBUG] Config: set publicIP(String) to x.x.x.x
 2023-08-14 12:13:57,255 [INFO] ICE: My public IP is x.x.x.x
 2023-08-14 12:13:57,255 [INFO] vncext: VNC extension running!
 2023-08-14 12:13:57,255 [DEBUG] Config: set immutable AllowOverride
 2023-08-14 12:13:57,256 [INFO] vncext: Listening for websocket connections on 0.0.0.0 interface(s), port 8444
 2023-08-14 12:13:57,256 [DEBUG] VNCServerST: creating single-threaded server freerdpdemokasm:1 (nx)
 2023-08-14 12:13:57,256 [INFO] VNCServerST: CPU capability: SSE2 yes, AVX512f no
 2023-08-14 12:13:57,256 [INFO] vncext: created VNC server for screen 0
 2023-08-14 12:13:57,320 [INFO] Input: Mouse horizonatl scroll factor: 50.000000
 2023-08-14 12:13:57,320 [INFO] Input: Mouse vertical scroll factor: 50.000000
 2023-08-14 12:13:57,361 [DEBUG] Selection: Selection owner change for _DBUS_SESSION_BUS_SELECTION_nx_cce08b3df2835e5385f9837ede898e0d
 2023-08-14 12:13:57,539 [DEBUG] WebUdp: UDP listening on port 8444
 2023-08-14 12:13:57,589 [DEBUG] Selection: Selection owner change for WM_S0
 2023-08-14 12:13:57,633 [DEBUG] Selection: Selection owner change for _NET_WM_CM_S0
 2023-08-14 12:13:57,633 [DEBUG] Selection: Selection owner change for COMPOSITING_MANAGER

(xfwm4:12035): xfwm4-WARNING **: 12:13:57.699: Unsupported GL renderer (llvmpipe (LLVM 15.0.7, 256 bits)).
 2023-08-14 12:13:57,699 [DEBUG] Selection: Selection owner change for XFWM4_COMPOSITING_MANAGER
Xvnc: ../../include/privates.h:121: dixGetPrivateAddr: Assertion `key->initialized' failed.
(EE) 
(EE) Backtrace:
(EE) 0: /usr/bin/Xvnc (OsLookupColor+0x139) [0x64f209]
(EE) 1: /lib/x86_64-linux-gnu/libc.so.6 (__sigaction+0x50) [0x7ff4db17356f]
(EE) 2: /lib/x86_64-linux-gnu/libc.so.6 (pthread_kill+0x12c) [0x7ff4db1c7a7c]
(EE) 3: /lib/x86_64-linux-gnu/libc.so.6 (raise+0x16) [0x7ff4db173476]
(EE) 4: /lib/x86_64-linux-gnu/libc.so.6 (abort+0xd3) [0x7ff4db1597f3]
(EE) unw_get_proc_name failed: no unwind info found [-10]
(EE) 5: /lib/x86_64-linux-gnu/libc.so.6 (?+0x0) [0x7ff4db15970c]
(EE) 6: /lib/x86_64-linux-gnu/libc.so.6 (__assert_fail+0x46) [0x7ff4db16ae96]
(EE) 7: /usr/bin/Xvnc (vncClientGone+0xe5) [0x46b575]
(EE) 8: /usr/bin/Xvnc (xvnc_sync_dri3_pixmap+0x209) [0x46b999]
(EE) 9: /usr/bin/Xvnc (present_copy_region+0x43) [0x4dea43]
(EE) 10: /usr/bin/Xvnc (present_execute_copy+0x73) [0x4df023]
(EE) 11: /usr/bin/Xvnc (present_set_abort_flip+0x1c7) [0x4e08b7]
(EE) 12: /usr/bin/Xvnc (present_fake_get_ust_msc+0xc7) [0x4df257]
(EE) 13: /usr/bin/Xvnc (TimerSet+0x180) [0x649180]
(EE) 14: /usr/bin/Xvnc (WaitForSomething+0x258) [0x6492a8]
(EE) 15: /usr/bin/Xvnc (Dispatch+0xb2) [0x5fbbc2]
(EE) 16: /usr/bin/Xvnc (dix_main+0x35a) [0x5ffe1a]
(EE) 17: /lib/x86_64-linux-gnu/libc.so.6 (__libc_init_first+0x90) [0x7ff4db15ada0]
(EE) 18: /lib/x86_64-linux-gnu/libc.so.6 (__libc_start_main+0x80) [0x7ff4db15ae40]
(EE) 19: /usr/bin/Xvnc (_start+0x25) [0x468585]
(EE) 
(EE) 
Fatal server error:
(EE) Caught signal 6 (Aborted). Server aborting
(EE) 
```

**System Description**
Provide the output of 
```
cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy

--

uname -a
Linux demokasm 5.15.0-78-generic #85-Ubuntu SMP Fri Jul 7 15:25:09 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

**KasmVNC Details**
[kasmvncserver_jammy_1.1.0_amd64.deb](https://github.com/kasmtech/KasmVNC/releases/download/v1.1.0/kasmvncserver_jammy_1.1.0_amd64.deb)

Provide the output of this command.
```
Xvnc -version

Xvnc KasmVNC 1.1.0.56c840fc947d891f054bf251d2a02454d3e6f686 - built Apr  5 2023 14:38:59
Copyright (C) 1999-2018 KasmVNC Team and many others (see README.me)
See http://kasmweb.com for information on KasmVNC.
Underlying X server release 12008000, The X.Org Foundation
```

**Comment by ne0YT** [comment]
this also happens when I do a reboot while there was still an open session.
I didn't find a fix yet

**Comment by mmcclaskey** [comment]
Would you mind trying out our release preview of 1.2.0?
https://kasmweb-build-artifacts.s3.amazonaws.com/kasmvnc/fa884809145e0ba6e5e1da49887a82b5cab0a1d1/kasmvncserver_jammy_1.1.1_master_fa8848_amd64.deb

**Comment by ne0YT** [comment]
@mmcclaskey 
yes, I tested, that fixes this issue!

**Comment by mmcclaskey** [comment]
https://kasmweb-build-artifacts.s3.amazonaws.com/kasmvnc/fa884809145e0ba6e5e1da49887a82b5cab0a1d1/kasmvncserver_focal_1.1.1_master_fa8848_amd64.deb

**Comment by ne0YT** [comment]
@mmcclaskey 

hey there I'm really sorry to cross-reference somethign which is absolutely not related..
but it is very urgent to me.
is this somewhat possible?:
https://github.com/kasmtech/KasmVNC/issues/171

**Comment by mmcclaskey** [comment]
Fixed in 1.2.0
